API/Extensions/UserExtensions.cs

```diff
@@ -0,0 +1,46 @@
+using API.DataModel.Entities.AspNetIdentity;
+using API.DTO;
+using System.Security.Claims;
+
+namespace API.Extensions
+{
+    public static class UserExtensions
+    {
+        public static EmployeeDto ToEmployeeDto(this User user)
+        {            
+            if (user!=null)
+            {
+                return new EmployeeDto
+                {
+                    Id = user.Id,
+                    FirstName = user.FirstName,
+                    LastName = user.LastName,
+                    PhoneNumber = user.PhoneNumber,
+                    Email = user.Email
+                };    
+            }
+            else            
+                return null;            
+        }
+        public static ClientDto ToClientDto(this User user)
```

Comment:
Z dobrych praktyk - tego nulla zwracasz świadomie czy podczas używania tej metody raczej nie chcialbyś sytuacji aby zwróciła null? Ja np. zamiast zwracać nulla to prawdopodobnie rzucałbym tu exception.

Reply:
Zwykle sprawdzałbym po stronie wywołującej ale nie wiem czy jest to dobra praktyka.  Jak widzisz w WWSI raczej nas tego nie uczyli jak wielu innych przydatnych rzeczy. Dzięki tym komentarzom widzę sporo niedoróbek w swojej pracy.